frontend/src/components/pages/CampsList/index.tsx

```diff
@@ -49,7 +49,7 @@ const CampsListPage = (): React.ReactElement => {
     if (!campToDelete) {
       return;
     }
-
+    onDeleteModalClose();
```

Comment:
This isn't for the waitlisted campers, but a similar problem exists if you try to delete a camp using the three dots beside a camp's name on the camplist (/admin route).